### ISSUE TITLE
Could anyone help me with the data loading process?

### ISSUE THREAD

**Issue by ShaneeyS** [NVlabs/GCVit#58]
Hi, 

When I try to train the model by myself, I have several questions about the ImageNet dataset. For ImageNet-1k, the validation data from official website is not a format of:

```
  imagenet
  ├── val
      ├── class1
      │   ├── img4.jpeg
      │   ├── img5.jpeg
      │   └── ...
      ├── class2
      │   ├── img6.jpeg
      │   └── ...
      └── ...
 
  ```

but the validation tar is formatted as:

```
  imagenet
  ├── val
      ├──  img1.jpeg
      ├──  img2.jpeg
      ├──  img3.jpeg
      └── ...

How to prepare the validation set into the desired format?


In addition, if the train set contains class0~class900, val set contains class900~1000. Should I create empty folders for train directory and test directory (so as to make each of them has 1000 folders) ?

Thanks a lot!

**Comment by ahatamiz** [comment]
Please see the following script to generate correct folders:
https://github.com/tensorflow/models/blob/master/research/slim/datasets/preprocess_imagenet_validation_data.py
